scanners/sslyze/parser/parser.test.js

```diff
@@ -32,24 +32,31 @@ test("parses result file for www.securecodebox.io correctly", async () => {
     hint: null,
     reference: null,
     location: "www.securecodebox.io:443",
-    attributes: {
+    attributes: {       
```

Comment:
This file has a bunch of whitespaces in weird places (see rest of the diff as well).

Reply:
My bad, I had to manually format a few parts. it should be fine now